environments/se/core/src/main/java/org/jboss/weld/environment/se/StartMain.java

```diff
@@ -52,7 +58,17 @@ public WeldContainer go() {
      * @param args the command line arguments
      */
     public static void main(String[] args) {
-        new StartMain(args).go();
+        try {
+            new StartMain(args).go();
+            System.exit(0);
+        } catch (Throwable t) {
+            StreamSupport.stream(exceptionHandlerLoader.spliterator(), false)
```

Comment:
I think there is no need to use streams here...

Reply:
I like the use of the ``Stream`` here but agree that it's a bit out of place in the ``ExceptionHandler`` interface.